crud/select/executor.lua

```diff
@@ -31,52 +30,51 @@ local function scroll_to_after_tuple(gen, param, state, space, scanner)
             return nil
         end
 
-        if scroll_comparator(tuple, scanner.after_tuple) then
+        if scroll_comparator(tuple, after_tuple) then
             return tuple
         end
     end
 end
 
-function executor.execute(plan)
-    local scanner = plan.scanner
+function executor.execute(space, index, scan_value, iter, filter_func, opts)
+    checks('table', 'table', '?table', 'number', 'function', {
+        after_tuple = '?cdata|table',
+        batch_size = '?number',
```

Comment:
number|cdata

Reply:
Or even `number|uint64|int64 `

Reply:
So, not sure checks really needed here it's internal module

Reply:
Yes, but it's very helpful for local development.
Moreover, all functions have a lot of arguments and different opts.

Reply:
> very helpful for local development.

Then do something like:
```lua
local checks = require('checks')
if os.getenv('DEV') ~= '1' then
    checks = function() end
end
```

And run all your tests with "DEV=1" option.

Reply:
See #35.